tests/e2e/bento_server_http/tests/test_meta.py

```diff
@@ -26,8 +27,9 @@ async def test_api_server_meta(host: str) -> None:
     status, _, _ = await async_request("GET", f"http://{host}/docs.json")
     assert status == 200
     status, _, _ = await async_request("GET", f"http://{host}/readyz")
-    assert status == 200
+    assert status == 500
     status, _, body = await async_request("GET", f"http://{host}/metrics")
+    await asyncio.sleep(20)
```

Comment:
I suggest to make a separate test, call it `test_readiness` and remove the async_request to readyz
```python
@pytest.mark.asyncio
async def test_readiness(host: str):
    await async_request("GET", f"http://{host}/readyz", assert_status=500)
    await asyncio.sleep(10)
    status, _, _ = await async_request("GET", f"http://{host}/readyz")
    assert status == 200
```

Reply:
Thanks for catching that, Aaron. I'll make the update.